src/chi-vue/src/components/data-table/DataTableEmptyActionable.tsx

```diff
@@ -16,14 +16,12 @@ export default class DataTableEmptyActionable extends Vue {
   message = this.$props.content.message || defaultConfig.emptyActionable?.message;
 
   render() {
-    return (
-      <div>
-        <chi-icon icon={this.icon} color="grey"></chi-icon>
-        <span>
-          <a onClick={() => this._emitEmptyActionableLink()}>{this.message.actionLink}</a>
-          {this.message.text}
-        </span>
-      </div>
-    );
+    return [
```

Comment:
Use `<div>` as a parent to return an element with children as you cannot return multiple nodes.